lib/linter/compiled-pane.js

```diff
@@ -7,6 +7,8 @@ import { CompositeDisposable } from 'atom'
 import PaneItem from '../util/pane-item'
 import { open } from '../util/opener'
 
+var subs
```

Comment:
Make this a class prop instead of a global?

Reply:
Ok, it is now handled like that.